### PR TITLE
#[162950845] Index page

### PR DESCRIPTION
**Pull request by mdn-network**
#### What does this PR do?

-Creates Landing page--enables user to either sign up, sign in and view popukar meetup questions

#### Description of Task to be completed?

- Building the landing page using css and html
- Add links to homepage, questions, sign up and sign in

#### How should this be manually tested?
- Clone the repo
`$git clone https://github.com/Philipotieno/Questioner.git` 
- Cd into the directory
`$ cd Questione`
- Open index.html file using a browser
#### Any background context you want to provide?
-Doesn't implement the core functionality yet
#### What are the relevant pivotal tracker stories?
-[#162950845]https://www.pivotaltracker.com/n/projects/2235179)
#### Screenshots 
landing page

### CODE REVIEW

File: UI/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+			<p>Friday, Jan 04 11:00 AM</p>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+		</div>
+	</div>
+	<div class="footer">
+		<p>@Questioners 2018</p>
+	</div>
+
+</body>
+</html>
```
Comment (Ogollah) [inline UI/index.html:42]: need to add a new line at the end of a file

File: UI/style.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+	margin-right: auto;
+
+
+}
+#myImg {
+	width: 520px;
+	height: 250px;
+	border-radius: 20px;
+
+}
```
Comment (Ogollah) [inline UI/style.css:117]: new line

**Comment by cod3smith** [comment]
Nice work man.

**Comment by mdn-network** [comment]
Thank you @KelynPNjeri

**Comment by Siffersari** [comment]
@Philipotieno  Great job! Use [feature <story Id> ] instead of [ ft <story Id>] when making your commits.

**Comment by mdn-network** [comment]
@Siffersari  implementing that from now onwards
Thanks bro

**Pull request merged**